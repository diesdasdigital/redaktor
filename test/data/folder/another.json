{
  "default": {
    "required": {
      "title": "Another",
      "pageType": "default"
    },
    "fields": {},
    "components": []
  }
}
