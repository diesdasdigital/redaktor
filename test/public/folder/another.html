{
  "markdown": "Another\n=======\n\n## File\n",
  "path": "folder/another",
  "updatedAt": "2020-08-28T12:38:38.009Z",
  "createdAt": "2020-08-28T12:38:38.009Z"
}