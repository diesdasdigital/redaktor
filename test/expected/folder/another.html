<pre>{
  "markdown": "Another\n=======\n\n## File\n",
  "path": "folder/another"
}</pre>